packages/functional-tests/lib/metrics.ts

```diff
@@ -0,0 +1,120 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Page } from '@playwright/test';
+
+const GlobalEventProperties = [
+  'country_code',
+  'event_type',
+  'language',
+  'country_code_source',
+  'time',
+  'user_id',
+];
+
+const SubscriptionCreateEventProperties = [
+  'entrypoint',
+  'entrypoint_experiment',
+  'entrypoint_variation',
+  'form_type',
+  'utm_campaign',
+  'utm_content',
+  'utm_term',
+  'utm_medium',
+  'context',
+  'checkout_type',
+  'error_id',
+  'flow_id',
+  'payment_provider',
+  'plan_id',
+  'product_id',
+  'promotion_code',
+  'subscribed_plan_ids',
+];
```

Comment:
A lot of these fields aren't unique to subscription create events (e.g. the acquisition query params) and in general I think some of these fields are globals at least for payments server events (e.g. `flow_id`). Let's sync on these together tomorrow.